spec/oas_parser/definition_spec.rb

```diff
@@ -76,4 +76,14 @@
       end
     end
   end
+
+  describe '#webhooks' do
+    it 'handles specs with no webhooks' do
+      expect(@definition.webhooks.count).to eq(0)
+    end
```

Comment:
nitpick: missing newline

Reply:
I'll merge, then add Rubocop to this repo and fix all violations